src/main/java/eu/arrowhead/blacklist/api/mqtt/ManagementMqttHandler.java

```diff
@@ -0,0 +1,104 @@
+package eu.arrowhead.blacklist.api.mqtt;
+
+import java.security.InvalidParameterException;
+import java.util.List;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.stereotype.Service;
+import org.springframework.util.Assert;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+
+import eu.arrowhead.blacklist.BlacklistConstants;
+import eu.arrowhead.blacklist.service.ManagementService;
+import eu.arrowhead.common.Constants;
+import eu.arrowhead.common.exception.ArrowheadException;
+import eu.arrowhead.common.mqtt.MqttStatus;
+import eu.arrowhead.common.mqtt.handler.MqttTopicHandler;
+import eu.arrowhead.common.mqtt.model.MqttRequestModel;
+import eu.arrowhead.dto.BlacklistCreateListRequestDTO;
+import eu.arrowhead.dto.BlacklistEntryListResponseDTO;
+import eu.arrowhead.dto.BlacklistQueryRequestDTO;
+
+@Service
+@ConditionalOnProperty(name = Constants.MQTT_API_ENABLED, matchIfMissing = false)
+public class ManagementMqttHandler extends MqttTopicHandler {
+
+	//=================================================================================================
+	// members
+
+	@Autowired
+	private ManagementService managementService;
+
+	private final Logger logger = LogManager.getLogger(getClass());
+
+	//=================================================================================================
+	// methods
+
+	//-------------------------------------------------------------------------------------------------
+	@Override
+	public String baseTopic() {
+		return BlacklistConstants.MQTT_API_MANAGEMENT_BASE_TOPIC;
+	}
+
+	//-------------------------------------------------------------------------------------------------
+	@Override
+	public void handle(final MqttRequestModel request) throws ArrowheadException {
+		logger.debug("ManagementMqttHandler.handle started");
+		Assert.isTrue(request.getBaseTopic().equals(baseTopic()), "MQTT topic-handler mismatch");
+		MqttStatus responseStatus = MqttStatus.OK;
+		Object responsePayload = null;
+
+		switch (request.getOperation()) {
+		case Constants.SERVICE_OP_BLACKLIST_QUERY:
+			final BlacklistQueryRequestDTO queryDto = readPayload(request.getPayload(), BlacklistQueryRequestDTO.class);
+			responsePayload = query(queryDto);
+			break;
+
+		case Constants.SERVICE_OP_BLACKLIST_CREATE:
+			final BlacklistCreateListRequestDTO createDto = readPayload(request.getPayload(), BlacklistCreateListRequestDTO.class);
```

Comment:
should return MqttStatus.OK (201). 

Also this is a problem in the HTTP service. The Swagger annotation tells us it returns with 201, but actually returns with 200. Should returns with 201.